phpBB/includes/functions_display.php

```diff
@@ -738,6 +738,11 @@ function generate_forum_rules(&$forum_data)
 		$forum_data['forum_rules'] = generate_text_for_display($forum_data['forum_rules'], $forum_data['forum_rules_uid'], $forum_data['forum_rules_bitfield'], $forum_data['forum_rules_options']);
 	}
 
+	if (!$forum_data['forum_rules'])
+	{
+		return;
```

Comment:
As far as I can see, this used to display the forum rules if rules_link is set. Maybe just check
```if ((!$forum_data['forum_rules'] || $forum_data['forum_rules'] === '<t></t>') && !$forum_data['forum_rules_link'])```
instead of modifying the logic here. This will now also return if forum link is set and forum rules isn't. Something it didn't use to do previously.